Backend/Dev/packages.config

```diff
@@ -1,4 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="EntityFramework" version="6.2.0" targetFramework="net462" />
+  <package id="EntityFramework" version="6.2.0" targetFramework="net461" />
```

Comment:
Same as above. I'm going to stop commenting on this issue, please check all other instances of this change.